webcomponents/tool/src/main/frontend/packages/sakai-rubrics/src/SakaiRubricCriteria.js

```diff
@@ -571,7 +580,7 @@ export class SakaiRubricCriteria extends RubricsElement {
                 <span>${this.maxPoints}</span>
               </div>
             </div>
-            <button class="btn-link save-weights" @click="${this.saveWeights}" ?disabled="${!this.validWeight && !this.isDraft}">
+            <button class="btn-link save-weights" @click="${this.saveWeights}" .disabled=${this._savingWeights || (!this.validWeight && !this.isDraft)}>
```

Comment:
it's a draft but the weights are incorrect: so you can save it? seems wrong to me.

Reply:
I agree

Reply:
It seems like it should be allowed if the weight isn't valid as long as it is a draft. But, why bother with the draft check? Why allow a save of invalid weights when they're so easy to update?

Reply:
It's intentional. If you save the rubrics as draft it forces the save of the criterion weights, and even gives you a tooltip telling you that. So, I don't want to mess with that. I agree it seems odd.